Error installing pybindings

When I try to install Quizx with its Python bindings, I get the following error:

```
error[E0308]: mismatched types
   --> src/lib.rs:231:36
    |
231 |     fn max_terms(&self) -> usize { self.d.max_terms() }
    |                            -----   ^^^^^^^^^^^^^^^^^^ expected `usize`, found `f64`
    |                            |
    |                            expected `usize` because of return type

```

Here is an exact description of what I tried, in a reproducible Dockerfile:

```Dockerfile
FROM rust:alpine3.16

#################
# PREREQUISITES #
#################

RUN apk update && \
    apk add \
        git \
        build-base \
        python3 \
        py3-pip

RUN pip3 install setuptools-rust

#################
# INSTALL QUIZX #
#################

RUN git clone https://github.com/Quantomatic/quizx.git /quizx

RUN cd /quizx/quizx && \
    cargo build && \
    cargo test

###########################
# INSTALL PYTHON BINDINGS #
###########################

RUN cd /quizx/pybindings && \
    python3 setup.py install

RUN cd /quizx/pybindings && \
    python3 setup2.py install
```

The error occurs for the last line.

--- a/pybindings/src/lib.rs
+++ b/pybindings/src/lib.rs
@@ -228,7 +228,7 @@ impl Decomposer {
         else { self.d.with_simp(quizx::decompose::SimpFunc::NoSimp); }
     }
 
-    fn max_terms(&self) -> usize { self.d.max_terms() }
+    fn max_terms(&self) -> f64 { self.d.max_terms() }
     fn decomp_top(&mut self) { self.d.decomp_top(); }
     fn decomp_all(&mut self) { self.d.decomp_all(); }
     fn decomp_until_depth(&mut self, depth: usize) { self.d.decomp_until_depth(depth); }

